campaignion_tracking/campaignion_tracking.module

```diff
@@ -119,13 +120,19 @@ function campaignion_tracking_webform_redirect_alter($redirect, $submission) {
     webform_paymethod_select_webform_submission_load($submissions);
 
     // Track a donation success (ds).
-    $fragments[] = 't=ds';
+    $trackingFragments[] = 't=ds';
     foreach ($submissions[$submission->sid]->payments as $payment) {
       // Add context for donation tracking.
-      $fragments[] = "d:m=" . $payment->method->name;
+      $donationFragments[] = "m=" . $payment->method->name;
     }
   }
 
+  $fragments[] = 't:'. implode('&', $trackingFragments);
+  $fragments[] = 'w:'. implode('&', $webformFragments);
+  if ($donationFragments) {
+    $fragments[] = 'd:'. implode('&', $donationFragments);
+  }
```

Comment:
Why do we use 3 prefixes for a single module?

Reply:
this was the original design which I left untouched for the moment... idea was to have different prefixes for different "sources"...

happy to use just one `t` namespace as well if this is more appreciated :)

Reply:
Should we refactor this now? (I guess now is better than "later"...)

Reply:
How about `w:title=One: & two; & three,`?

Reply:
I’m not sure whether now is the best time. The original intention was to give each module (or use-case) one prefix so that they don’t interfere with each other.